psignifit/result.py

```diff
@@ -0,0 +1,52 @@
+import dataclasses
+from typing import Any, Dict, TextIO, Union
+import json
+from pathlib import Path
+
+import numpy as np
+
+from .configuration import Configuration
+
+
+class NumpyEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, np.ndarray):
+            return obj.tolist()
+        return json.JSONEncoder.default(self, obj)
+
+
+@dataclasses.dataclass(frozen=True)
+class Result:
+    sigmoid_parameters: Dict[str, float]
+    configuration: Configuration
+
+    @classmethod
+    def from_dict(cls, result_dict: Dict[str, Any]):
+        result_dict = result_dict.copy()
+        config_dict = result_dict.pop('configuration')
+        # Numpy arrays should be initialized from lists here.
```

Comment:
is this comment a relic from the past or is some code missing here?

Reply:
I made this more clear.